dev/com.ibm.ws.jca_fat_example_anno/fat/src/test/examplera/anno/FATSuite.java

```diff
@@ -10,24 +10,15 @@
  *******************************************************************************/
 package test.examplera.anno;
 
-import org.junit.ClassRule;
 import org.junit.runner.RunWith;
 import org.junit.runners.Suite;
 import org.junit.runners.Suite.SuiteClasses;
 
-import componenttest.rules.repeater.EmptyAction;
-import componenttest.rules.repeater.FeatureReplacementAction;
-import componenttest.rules.repeater.RepeatTests;
+import componenttest.custom.junit.runner.AlwaysPassesTest;
 
 @RunWith(Suite.class)
-@SuiteClasses({
-                ResourceAdapterExampleTest.class
-})
+@SuiteClasses({ AlwaysPassesTest.class,
+                ResourceAdapterExampleTest.class })
 public class FATSuite {
-    /*
-     * EE7 will run with full fat only. EE9 will be run with lite and full fat.
-     */
-    @ClassRule
-    public static RepeatTests r = RepeatTests.with(new EmptyAction().fullFATOnly())
-                    .andWith(FeatureReplacementAction.EE9_FEATURES());
+    //this tests the EE10 connectors version 2.1 added generic support for MappedRecord, IndexedRecord and should not be repeated for previous versions
```

Comment:
Are we losing test coverage for previous JCA / Connectors versions?

Reply:
No, we discussed this in scrum and Nathan decided we should convert this project because there is duplicate test coverage for this function in other projects.